File: output/StarCoder/codenet/Python/Java/s486032847.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String N = scanner.nextLine();
        // String[] inputs = scanner.nextLine().split(" ");
        // int N = Integer.parseInt(inputs[0]);
        // int X = Integer.parseInt(inputs[1]);
        // int T = Integer.parseInt(inputs[2]);

        if (keta(N) % 9 == 0) {
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }

    public static int keta(String s) {
        if (s.length() < 2) {
            return Integer.parseInt(s);
        }
        int n = 0;
        for (char c : s.toCharArray()) {
            n += Character.getNumericValue(c);
        }
        return keta(String.valueOf(n));
    }
}

File: dataset_reduzido/codenet/Python/Code/s486032847.py
import math

def keta(s):
    if len(s) < 2:
        return int(s)
    n = sum(map(int, s))
    return keta(str(n))

def main():
    N = input()
    # N, X, T = list(map(int, input().split()))
    
    if keta(N) % 9 == 0:
        print("Yes")
    else:
        print("No")



if __name__ == "__main__":
    main()